### PR TITLE
add: `resticprofile-bin`

### PR DESCRIPTION
**Pull request by erenfro**
Configuration profiles manager for restic backup.

### CODE REVIEW

**Comment by erenfro** [comment]
This is fully tested and ready for merging.

File: packages/resticprofile-bin/resticprofile-bin.pacscript [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+source_ppc64el=("https://github.com/creativeprojects/resticprofile/releases/download/v${pkgver}/resticprofile_${pkgver}_linux_ppc64le.tar.gz")
+source_riscv64=("https://github.com/creativeprojects/resticprofile/releases/download/v${pkgver}/resticprofile_${pkgver}_linux_riscv64.tar.gz")
+source_s390x=("https://github.com/creativeprojects/resticprofile/releases/download/v${pkgver}/resticprofile_${pkgver}_linux_s390x.tar.gz")
+pkgdesc="Configuration profiles manager for restic backup"
+url="https://github.com/creativeprojects/resticprofile"
+maintainer=("Eric Renfro <psi-jack@linux-help.org>")
+
+package() {
+  cd "${_archive}"
+  # Install binary
```
Comment (Elsie19) [inline packages/resticprofile-bin/resticprofile-bin.pacscript:28]:
```suggestion
```

File: packages/resticprofile-bin/resticprofile-bin.pacscript [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+source_arm64=("https://github.com/creativeprojects/resticprofile/releases/download/v${pkgver}/resticprofile_${pkgver}_linux_arm64.tar.gz")
+source_arm=("https://github.com/creativeprojects/resticprofile/releases/download/v${pkgver}/resticprofile_${pkgver}_linux_armv7.tar.gz")
+source_i386=("https://github.com/creativeprojects/resticprofile/releases/download/v${pkgver}/resticprofile_${pkgver}_linux_386.tar.gz")
+source_mips64el=("https://github.com/creativeprojects/resticprofile/releases/download/v${pkgver}/resticprofile_${pkgver}_linux_mips64le.tar.gz")
+source_ppc64el=("https://github.com/creativeprojects/resticprofile/releases/download/v${pkgver}/resticprofile_${pkgver}_linux_ppc64le.tar.gz")
+source_riscv64=("https://github.com/creativeprojects/resticprofile/releases/download/v${pkgver}/resticprofile_${pkgver}_linux_riscv64.tar.gz")
+source_s390x=("https://github.com/creativeprojects/resticprofile/releases/download/v${pkgver}/resticprofile_${pkgver}_linux_s390x.tar.gz")
+pkgdesc="Configuration profiles manager for restic backup"
+url="https://github.com/creativeprojects/resticprofile"
```
Comment (Elsie19) [inline packages/resticprofile-bin/resticprofile-bin.pacscript:23]:
```suggestion
url='https://github.com/creativeprojects/resticprofile'
```

**Review by Elsie19 — CHANGES_REQUESTED** [review]

**Review by Elsie19 — APPROVED** [review]

**Review by Elsie19 — DISMISSED** [review]

**Comment by Elsie19** [comment]
Oops, I did it twice 😁

**Comment by Elsie19** [comment]
Fix the merge conflicts then I can merge.

**Pull request closed without merging**